Manuel/Challenges/temperature.py

```diff
@@ -0,0 +1,18 @@
+
+# Convert a temperature of 100 degrees fahrenheit to celsiu
+# Save this to a variable called celsius_100, and use print() to print out the value
+# Is the resulting temperature you get an integer or float? How do you know?
+celsius_100 =(100-32)*5/9
+print(celsius_100)
+
+# Convert a temperature of 0 degrees fahrenheit to celsius
+# Save this to a variable called celsius_0, and use print() to print out the value
+celsius_0 =(0-32)*5/9
+print(celsius_0)
+
+# Convert a temperature of 34.2 degrees fahrenheit to celsius
+# Do this one all in one print statement without saving any variables
+print((5*9/5)+32)
+f_30_point_2 = (30.2*9/5)+32
+print(f_30_point_2)
+print(f(30.2 degrees celsius is equal to {f_30_point_2} degrees fahrenheit, its hotter than 85.1 degrees fahrenheit))
```

Comment:
think of how easy a challenge such as this is now. You've come a long way. Keep up the great work